File: courzelo-dashboard-LMS-main/src/app/shared/models/institution/GroupResponse.ts
import {SimplifiedClassRoomResponse} from './SimplifiedClassRoomResponse';
import {SimplifiedProgramResponse} from './SimplifiedProgramResponse';

export interface GroupResponse {
    id?: string;
    name?: string;
    institutionID?: string;
    students?: string[];
    classrooms?: SimplifiedClassRoomResponse[];
    program?: string;
    simplifiedProgram?: SimplifiedProgramResponse;
}

File: Courzelo-backend-LMS/Courzelo/src/main/java/org/example/courzelo/dto/responses/GroupResponse.java
package org.example.courzelo.dto.responses;

import lombok.Builder;
import lombok.Data;
import org.example.courzelo.dto.responses.institution.SimplifiedClassRoomResponse;
import org.example.courzelo.models.Timetable.Semester;

import java.util.List;

@Data
@Builder
public class GroupResponse {
    private String id;
    private String name;
    private String institutionID;
    private List<String> students;
    private String program;
    private List<SimplifiedClassRoomResponse> classrooms;
    private List<Semester>semesters;
}
